features/step_definitions/work_download_steps.rb

```diff
@@ -1,3 +1,21 @@
+When /^I download "(.*?)"$/ do |title|
```

Comment:
Sorry, one more thing! This new step should also take the download format as an argument (using [Cucumber expressions](https://github.com/cucumber/cucumber-expressions#readme) rather than a regular expression) so we can use it in all of the other scenarios that currently do: 

```
I view the work "Work Title"
I follow "HTML/EPUB/Whatever"
```

Or, even easier, we can just use those two steps in the new tests as well.

No rush, though! We're saving up all of the download-related issues for [a single release](https://otwarchive.atlassian.net/issues/?filter=10601&jql=project%20%3D%20AO3%20AND%20fixVersion%20%3D%20%220.9.3xx%20(Downloads)%22), and there are a lot that still need fixing before we're ready. (All of our download files are cached, so we're going to have dump the cache to ensure the fixes get applied across the board, not just to new works, and we'd prefer to do that as infrequently as possible.)

Reply:
No worries! I've updated to use the existing steps. I was planning to just go through that release (either either 1 issue per PR or grouping similar issues together, whichever is preferred) next, but if you'd prefer I can go back to the wiki page.